junit5/src/main/java/cz/xtf/junit5/extensions/SkipForCondition.java

```diff
@@ -51,11 +51,12 @@ public static ConditionEvaluationResult resolve(SkipFor skipFor) {
         } else if (!skipFor.imageMetadataLabelName().equals("")) {
             DockerImageMetadata metadata = DockerImageMetadata.get(OpenShifts.master(), image);
             matcher = Pattern.compile(skipFor.imageMetadataLabelName()).matcher(metadata.labels().get("name"));
-        } else {
+        } else if (!skipFor.imageMetadataLabelArchitecture().equals("")) {
             DockerImageMetadata metadata = DockerImageMetadata.get(OpenShifts.master(), image);
             matcher = Pattern.compile(skipFor.imageMetadataLabelArchitecture()).matcher(metadata.labels().get("architecture"));
+        } else {
+            matcher = Pattern.compile(skipFor.subId()).matcher(XTFConfig.get("xtf.eap.subid"));
```

Comment:
It needs to be somehow generic, not using `eap` id.

Reply:
Thanks @simkam ! I must have lost it somewhere, fixing!